resources/istio-configuration/templates/istio-operator.yaml

```diff
@@ -24,7 +24,8 @@ spec:
       k8s:
 {{- toYaml .Values.components.pilot.config | nindent 10}}
   meshConfig:
-{{- toYaml .Values.meshConfig | nindent 4 }}
+{{- tpl (toYaml .Values.meshConfig | nindent 4) . | replace "'true'" "true" | replace "'false'" "false" }}
```

Comment:
what if it'll be "true" or `true` instead of 'true'

Reply:
If I understand correctly difference in Yaml between singl and double quotes it won't happen because this needs to be evaluated as string. Double quotes will attempt to evaluate special characters such as `\n` as new line instead just printing `\n` string

Reply:
I meant that it probably won't be replaced to boolean: replace "'true'" "true", will catch only `'true'`, but not `"true"`

Reply:
I agree but imo we don't need to care for this because it won't evaluate to `"true"` with double quotes.

Reply:
Perhaps I don't get your intention here, but it doesn't work for me:
```
> istio-configuration % helm template istio . --set global.tracing.enabled="'true'" | grep enableTracing
    enableTracing: 'true'
```
As you can see, the resulting value is a string: `'true'`, not a boolean `true`